code/datums/diseases/advance/symptoms/neurological.dm

```diff
@@ -0,0 +1,81 @@
+/datum/symptom/nerve_overdrive
+	name = "Neurological Overdrive"
+	desc = "The disease increases connectivity between the infected's nerves, reducing stuns. Increased nerve connections however make burns far more painful."
```

Comment:
Stun resistance is one thing I absolutely did not want to give virology, i dont want them able to just become entirely unstoppable in combat with a nice combat disease. Especially if you're adding a movement speed threshold here.

Reply:
It's arguably worse than hulk or meth when mixed with mannitol. Your resistance to burn on each body part is reduced by 2.5. So even if you have Biometallic, you are at a net negative for burn resistance.